db/views/reporting_patient_follow_ups_v01.sql

```diff
@@ -0,0 +1,75 @@
+WITH
+follow_up_blood_pressures AS (
+  SELECT DISTINCT ON (patient_id, facility_id, user_id, month_string)
+    p.id AS patient_id,
+    bp.facility_id,
+    bp.user_id,
+    to_char(bp.recorded_at AT TIME ZONE 'UTC' AT TIME ZONE (SELECT current_setting('TIMEZONE')), 'YYYY-MM') AS month_string
+  FROM patients p
+  INNER JOIN blood_pressures bp
+    ON p.id = bp.patient_id
+    -- removing bps that were recorded the same month as registration
+    AND to_char(bp.recorded_at AT TIME ZONE 'UTC' AT TIME ZONE (SELECT current_setting('TIMEZONE')), 'YYYY-MM')
+      > to_char(p.recorded_at AT TIME ZONE 'UTC' AT TIME ZONE (SELECT current_setting('TIMEZONE')), 'YYYY-MM')
+),
```

Comment:
This is the only subquery that's being used right now, the others are in preparation for upcoming follow up adjustments.